package.json

```diff
@@ -3559,23 +3559,23 @@
                 "command": "aws.codeWhisperer",
                 "key": "alt+c",
                 "mac": "alt+c",
-                "when": "editorTextFocus && CODEWHISPERER_ENABLED"
+                "when": "editorTextFocus && (CODEWHISPERER_ENABLED || isCloud9)"
             },
             {
                 "command": "aws.codeWhisperer.rejectCodeSuggestion",
```

Comment:
The actual function in this command needs to check `AuthUtil.instance.isConnectionValid()` right? Since in C9 this will always be enabled